barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/verifier/verifier.test.cpp

```diff
@@ -180,6 +180,38 @@ template <typename BuilderType> class RecursiveVerifierTest : public testing::Te
             EXPECT_EQ(recursive_manifest[i], native_manifest[i]);
         }
     }
+
+    /**
+     * @brief Construct a verifier circuit for a proof whose data has been tampered with. Expect failure
+     * TODO(#656): For now we get a "bad" proof by arbitrarily tampering with bits in a valid proof. It would be
```

Comment:
Please add this to the github issue as well so it's not forgotten, also can you point to the bberg issue, if it's just a number, it may be associated with aztec-package issue.

Reply:
done